src/state/reducers/index.js

```diff
@@ -8,6 +8,7 @@ import { reducer as tasks } from './taskReducer';
 import { reducer as team } from './teamReducers';
 import { reducer as faceoffs } from './faceoffsReducer';
 import { reducer as votes } from './votesReducer';
+import { reducer as squad } from './squadReducer';
```

Comment:
the faceoffs reducer is the same as the squad reducer, theyre just named differently in the 2 different repos, this is redundant

Reply:
ahh ok. I can use the faceoffs reducer..